.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: AwesomeBot
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
@nathany yes, there's no need for parameters here. https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#workflow_dispatch
In a short version this can look like this:
```yaml
on: workflow_dispatch
```
but there are other events here (`push` and `pull_request`), so it needs to be used in that way.

Most of the time workflows will be triggered automatically for pushes to master or for pull requests to master.
However I added a possibility to trigger workflow manually on any branch without making a pull request.
If you want, I can remove it. It's not crucial to have it here.

Reply:
@nathany yes, there's no need for parameters here. https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#workflow_dispatch
In a short version this could look like this:
```yaml
on: workflow_dispatch
```
but there are other events here (`push` and `pull_request`), so it needs to be used in that way.

Most of the time workflows will be triggered automatically for pushes to master or for pull requests to master.
However I added a possibility to trigger workflow manually on any branch without making a pull request.
If you want, I can remove it. It's not crucial to have it here.

Reply:
I decided to remove workflow_dispatch from this pull request to keep Github Actions configuration simpler.